content/es/docs/concepts/architecture/leases.md

```diff
@@ -0,0 +1,103 @@
+descubrir cuántas instancias de `kube-apiserver` están operando el plano de control de Kubernetes.
+La existencia de los objetos leases de kube-apiserver permite futuras capacidades que pueden requerir la coordinación entre
+cada kube-apiserver.
+
+Puede inspeccionar los leases de cada kube-apiserver buscando objetos leases en el espacio de nombres `kube-system`
+con el nombre `kube-apiserver-<sha256-hash>`. También puede utilizar el selector de etiquetas `apiserver.kubernetes.io/identity=kube-apiserver`:
+
+```shell
+kubectl -n kube-system get lease -l apiserver.kubernetes.io/identity=kube-apiserver
+```
+```
+NAME                                        HOLDER                                                                           AGE
+apiserver-07a5ea9b9b072c4a5f3d1c3702        apiserver-07a5ea9b9b072c4a5f3d1c3702_0c8914f7-0f35-440e-8676-7844977d3a05        5m33s
+apiserver-7be9e061c59d368b3ddaf1376e        apiserver-7be9e061c59d368b3ddaf1376e_84f2a85d-37c1-4b14-b6b9-603e62e4896f        4m23s
+apiserver-1dfef752bcb36637d2763d1868        apiserver-1dfef752bcb36637d2763d1868_c5ffa286-8a9a-45d4-91e7-61118ed58d2e        4m43s
+
+```
+El hash SHA256 utilizado en el nombre del lease se basa en el nombre de host del sistema operativo visto por ese servidor API. Cada kube-apiserver debe ser
+configurado para utilizar un nombre de host que es único dentro del clúster. Las nuevas instancias de kube-apiserver que utilizan el mismo nombre de host
+asumirán los leases existentes utilizando una nueva identidad de titular, en lugar de instanciar nuevos objetos leases. Puede comprobar el
+nombre de host utilizado por kube-apiserver comprobando el valor de la etiqueta `kubernetes.io/hostname`:
+
+```shell
+kubectl -n kube-system get lease apiserver-07a5ea9b9b072c4a5f3d1c3702 -o yaml
+```
+```yaml
+apiVersion: coordination.k8s.io/v1
+kind: Lease
+metadata:
+  creationTimestamp: "2023-07-02T13:16:48Z"
+  labels:
+    apiserver.kubernetes.io/identity: kube-apiserver
+    kubernetes.io/hostname: master-1
+  name: apiserver-07a5ea9b9b072c4a5f3d1c3702
+  namespace: kube-system
+  resourceVersion: "334899"
+  uid: 90870ab5-1ba9-4523-b215-e4d4e662acb1
+spec:
+  holderIdentity: apiserver-07a5ea9b9b072c4a5f3d1c3702_0c8914f7-0f35-440e-8676-7844977d3a05
+  leaseDurationSeconds: 3600
+  renewTime: "2023-07-04T21:58:48.065888Z"
+```
+
+Los leases caducados de los kube-apiservers que ya no existen son recogidos por los nuevos kube-apiservers después de 1 hora.
+
+Puede desactivar el lease de identidades del servidor API desactivando la opción `APIServerIdentity` de la [puerta de función](/docs/reference/command-line-tools-reference/feature-gates/).
+
+## Cargas de trabajo {#custom-workload}
+
+Su propia carga de trabajo puede definir su propio uso de los leases. Por ejemplo, puede ejecutar un
+{{< glossary_tooltip term_id=«controller» text=«controlador» >}} en la que un miembro principal o líder
```

Comment:
The PR build failed [(logs here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6629467a76432200089f88b7) encountering an error in this line as mentioned below. 

[Build Logs](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6629467a76432200089f88b7)

> 6:53:07 PM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
6:53:07 PM: ERROR [es] "docs/concepts/architecture/leases.md": "«controller»" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list


Please take a look at the suggestion provided. I hope this helps.
```suggestion
{{< glossary_tooltip term_id=controller text=controlador >}} en la que un miembro principal o líder
```